developer_manual/design/navigation.rst

```diff
@@ -226,7 +226,7 @@ Do not change the alignment of the text. If you're using
 
     <li class="app-navigation-entry-utils-counter">1</li>
 
-The count should be limitted to 999 and turn to 999+ if any higher number is given. If AngularJS is used the following filter can be used to get the correct behavior:
+The counter should be limited to 999 and turn to 999+ if any higher number is given. If AngularJS is used the following filter can be used to get the correct behavior:
```

Comment:
@nickvergessen 99+ instead of 999+, or am I wrong? https://github.com/nextcloud/spreed/pull/845

Reply:
No, you read the comments wrong

Reply:
I see. (At least I think so 😅)